CRM/Contribute/Form/UpdateSubscription.php

```diff
@@ -155,13 +155,17 @@ public function buildQuickForm() {
       TRUE, 'currency', $this->_subscriptionDetails->currency, TRUE
     );
 
+    // https://lab.civicrm.org/dev/financial/-/issues/197 https://github.com/civicrm/civicrm-core/pull/23796
+    // Revert freezing on total_amount field on recurring form - particularly affects IATs
+    // This will need revisiting in the future as updating amount on recur does not work for multiple lineitems.
```

Comment:
This is fine - but I don't actually see us going back to freezing it for single line items - rather I think we should update the template contribution when it is edited for single line items.